app/uk/gov/hmrc/eusubsidycompliancefrontend/services/SendEmailHelperService.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.services
+
+import play.api.Configuration
+import play.api.i18n.I18nSupport.RequestWithMessagesApi
+import play.api.i18n.MessagesApi
+import uk.gov.hmrc.eusubsidycompliancefrontend.actions.requests.EscAuthRequest
+import uk.gov.hmrc.eusubsidycompliancefrontend.config.AppConfig
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Language
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Language.{English, Welsh}
+import com.google.inject.Inject
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Undertaking
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.email.EmailParameters.{DoubleEORIAndDateEmailParameter, DoubleEORIEmailParameter, SingleEORIAndDateEmailParameter, SingleEORIEmailParameter}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.email.{EmailParameters, EmailSendResult}
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.{EORI, UndertakingRef}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import java.util.Locale
+import javax.inject.Singleton
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SendEmailHelperService @Inject() (
+    appConfig: AppConfig,
+    retrieveEmailService: RetrieveEmailService,
+    sendEmailService: SendEmailService,
+    configuration: Configuration) {
+
+
+  def retrieveEmailAddressAndSendEmail(eori1: EORI,
```

Comment:
Could we not just move this into the SendEmailService? Seems unnecessary to have two services around email.